addons/html_editor/static/tests/insert/paragraph_break.test.js

```diff
@@ -330,7 +330,10 @@ describe("Selection collapsed", () => {
             });
         });
 
-        test("should insert line breaks outside the edges of an anchor", async () => {
+        // skipping these tests cause with the link isolation the cursor can be put
+        // inside/outside the link so the user can choose where to insert the line break
+        // see `anchor.nodeName === "A" && brEls.includes(anchor.firstChild)` in line_break_plugin.js
+        test.skip("should insert line breaks outside the edges of an anchor", async () => {
```

Comment:
This test isn't valid anymore ?

Reply:
These tests are not valid anymore because we removed the code which should only be done for unbreakable in this PR. But we were not sure if Antoine did it for some special cases in mass mailing or website, cause it was't clear on the original [PR](https://github.com/odoo/odoo/pull/81662). So we marked them as skipped in case there's a use case we really need to do this for unbreakable.

The reason we are removing it is that after we have the link isolation, the user can choose if the cursor should be placed inside of a link `<a>ABC[]</a>` or outside of a link `<a>ABC</a>[]`, so we think taking the `<br>` outside of a link is not necessary